pkg/fixer/fixes/nowhitespacecomment.go

```diff
@@ -0,0 +1,43 @@
+package fixes
+
+import (
+	"bytes"
+)
+
+type NoWhitespaceComment struct{}
+
+func (*NoWhitespaceComment) Key() string {
+	return "no-whitespace-comment"
+}
+
+func (*NoWhitespaceComment) WholeFile() bool {
+	return false
+}
+
+func (*NoWhitespaceComment) Fix(in []byte, opts *RuntimeOptions) (bool, []byte, error) {
+	lines := bytes.Split(in, []byte("\n"))
+
+	// this fix must have locations
+	if len(opts.Locations) == 0 {
+		return false, nil, nil
+	}
+
+	for _, loc := range opts.Locations {
+		if loc.Row > len(lines) {
+			return false, nil, nil
+		}
+
+		if loc.Col != 1 {
+			// current impl only understands the first column
```

Comment:
Interesting. Just a shortcut or tricky to make it work if the comment is later on the line?

Reply:
See note below, fixed in https://github.com/StyraInc/regal/commit/fa62e1865e9484ca50014b86833b01b845fefa61